api/src/main/java/pl/kolendateam/dadcard/abilitys/entity/Abilitys.java

```diff
@@ -56,5 +56,19 @@ public Abilitys addRaceAbilitys(Abilitys jsonObjectAbilitys, Abilitys abilitys)
         
         return abilitys;
     }
+
+    public boolean checkPrerequisiteAb(Abilitys ab) {
```

Comment:
starajmy sie nie zrobic zmiennych ab, pg, tw itp. tak by widzac kod wiedziec  co to.

Reply:
nie wystarczy return  return streght >= ab.streght &&
            dextrity >= ab.dextrity &&
            constitution >= ab.constitution &&
            intelligence >= ab.intelligence &&
            wisdom >= ab.wisdom &&
            charisma >= ab.charisma ?

Reply:
tak, jest nie potrzebne return true/false, zmieniam też ab na abilitys, będę musiał generalnie na całym projekcie lepiej wszystkie zmienne opisać.